front/src/components/map/filter/PeriodFilterButton.vue

```diff
@@ -81,28 +79,12 @@ export default {
       this.$store.commit("post/filter/RESET_END_DATE");
       this.filterPosts();
     },
-    inputStartDate(date) {
-      this.startDate = period.format(date + " 00:00:00");
+    inputDates(dates) {
```

Comment:
```suggestion
    inputDates({startDate, endDate}) {
```

객체로 반환하면 조금 더 편하지 않을까요? dates[-]을 쓰는 것 보단 나을 것 같아요.😄